Resources/Prototypes/_Sunrise/Catalog/uplink_catalog.yml

```
@@ -381,16 +381,56 @@
+  - !type:StoreWhitelistCondition
+    whitelist:
+      tags:
+      - NukeOpsUplink
+
     Telecrystal: 3
+
+- type: listing
+  id: UplinkMagazineDragunov
+  # name: uplink-magazine-dragunov-name
+  description: uplink-magazine-dragunov-desc
+  icon: { sprite: _Sunrise/Objects/Weapons/Guns/Ammunition/Magazines/Rifle/dragunov_mag.rsi, state: base }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)